specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/examples/CreateOrUpdateAvailabilityGroupListener.json

```diff
@@ -8,6 +8,24 @@
     "parameters": {
       "properties": {
         "availabilityGroupName": "ag-test",
+        "availabilityGroupConfiguration": {
+          "replicas": [
+            {
+              "sqlVirtualMachineInstanceId": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.SqlVirtualMachine/sqlVirtualMachines/testvm2",
+              "role": "Primary",
```

Comment:
>Primary [](http://example.com/codeflow?start=23&length=7)

Please fix the modelValidation errors on every enum - there is a casing issue. 
Expected: [https://github.com/Azure/azure-rest-api-specs/blob/a4817b0863fa45d3cb995269d0d95516cf1a3862/specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/sqlvm.json#L1097:17](https://github.com/Azure/azure-rest-api-specs/blob/a4817b0863fa45d3cb995269d0d95516cf1a3862/specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/sqlvm.json#L1097:17)

Reply:
> > Primary
> 
> Please fix the modelValidation errors on every enum - there is a casing issue. Expected: https://github.com/Azure/azure-rest-api-specs/blob/a4817b0863fa45d3cb995269d0d95516cf1a3862/specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/sqlvm.json#L1097:17

Looks like the label was never updated, setting ARMChangesRequested now. +1 on the comment to fix the model validation errors. Also need to fix the merge errors.